src/stan-users-guide/part3.Rmd

```diff
@@ -1,4 +1,4 @@
-# <i style="font-size: 110%; color:#990017;">Part 3. Posterior Inference & Model Checking</i> {- #meta-models.part}
+# <i style="font-size: 110%; color:#052744;">Part 3. Posterior Inference & Model Checking</i> {.unnumbered #meta-models.part}
```

Comment:
defined here:  https://github.com/stan-dev/docs/blob/299e90e13240932edaaf4c8f034fbf6247f0fd55/src/quarto_styles.css#L3-L12.  (redundancies because I was trying different things)

Reply:
In that case I think you can use `color: var(--bs-body-color)` instead of the hex code, which would make any changes in the future easier to sync?